packages/react/src/experimental/OneTable/TableHead/index.tsx

```diff
@@ -118,15 +124,28 @@ export function TableHead({
     </>
   )
 
+  const isNumber = (width: ColumnWidth): width is number =>
```

Comment:
should we have this function shared? there is a duplication in the `TableCell` file